nikola/post.py

```diff
@@ -987,6 +1002,24 @@ def source_ext(self, prefix=False):
         else:
             return ext
 
+    def write_post(self):
+        """Do a best effort to write down the current post state, including metadata.
+
+        It will save in either 2-file or 1-file format, in the format indicated by
+        METADATA_FORMAT.
+
+        This has caveats:
+
+        * If you are extracting metadata from filename?
+        * If you are extracting metadata from document contents?
+
+        Because of how it's going to be written, that's not going to be used anymore.
+        """
+        breakpoint()
```

Comment:
Debugging leftover?

Reply:
I forgot to remove the started but-not-done method.